src/wmnds/components/content-tiles/travel-updates/travel-updates.njk

```diff
@@ -49,11 +49,11 @@
       <div class="wmnds-content-tile__container-bus">
          {{
             medium({
-            classModifiers: 'wmnds-disruption-indicator-medium--success wmnds-disruption-indicator-medium--narrow-travel-updates wmnds-disruption-indicator-medium--narrow-travel-updates-bus',
-            text: '12',
+            classes: 'wmnds-disruption-indicator-medium--success wmnds-disruption-indicator-medium--narrow-travel-updates wmnds-disruption-indicator-medium--narrow-travel-updates-bus',
```

Comment:
Make some of these show a disruption rather than them all being good(green) service.